### ISSUE TITLE
Fajok nevei a csillagtérképen és bolygóknál

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#23]
```
Game version:0.92

A fajoknál van 2 aminek a neve nem fér ki a bolygók listájánál, odaírom 
az eredeti játékban hogy írták ki, hogy kiférjen:

Szabad Nemzetek Szövetsége: Szabad Nemz. Sz.
Szabad Kereskedők Szövetsége: Kereskedők Szöv. (egyébként nincs előtte, 
hogy Szabad)

```

Original issue reported on code.google.com by `Jozsef.T...@gmail.com` on 16 Aug 2011 at 9:45

**Comment by GoogleCodeExporter** [comment]
```
Képernyőfotót is küldenél?
```

Original comment by `akarn...@gmail.com` on 16 Aug 2011 at 10:24

**Comment by GoogleCodeExporter** [comment]
```
Kicsit kilógnak a nevek :)
```

Original comment by `Jozsef.T...@gmail.com` on 16 Aug 2011 at 10:37

Attachments:
- [kereskedok.jpg](https://storage.googleapis.com/google-code-attachments/open-ig/issue-23/comment-2/kereskedok.jpg)
- [sznsz.jpg](https://storage.googleapis.com/google-code-attachments/open-ig/issue-23/comment-2/sznsz.jpg)

**Comment by GoogleCodeExporter** [comment]
```
Fix 0.93.100
```

Original comment by `akarn...@gmail.com` on 17 Aug 2011 at 11:05
- Changed state: **Fixed**
